src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -103,7 +104,14 @@ else if (modules.size() > 1) {
           LOG.warn(MessageFormat.format("{0} contains {1} modules.", project.getName(), modules.size()));
```

Comment:
It's not part of your change, but I'm wondering if this should really be a warning?
If there is more than one module, perhaps just return?